AndroidPubSub/build.gradle

```diff
@@ -21,6 +21,7 @@ dependencies {
     def aws_version = "2.9.+"
     implementation "com.amazonaws:aws-android-sdk-iot:$aws_version"
     implementation "com.amazonaws:aws-android-sdk-mobile-client:$aws_version"
+    implementation "com.android.support:support-compat:28.0.0-alpha1"
```

Comment:
28.0.0 is available.

Reply:
Okay. I picked it from their documentation which doesn't seem to have been updated yet. (https://developer.android.com/reference/android/support/v4/content/ContextCompat). Updated the dependency.